### PR TITLE
Add certificates

### PR DESCRIPTION
**Pull request by yelyzavetakhokhlova**
### PR Checklist

* [x] Have you verified that the PR is pointing to the correct target branch? (`develop` for features/bugfixes, `master` for release/hotfix)?
* [x] Have you followed the guidelines in our [Dev Guide](../#readme)
* [x] Have you run prettier (`npm run format`)?
* [x] Have you checked that the build output looks according to design (screen, mobile, desktop) in Figma (`npm run serve`)?

## Visuals
image
image
image
image

### CODE REVIEW

File: src/components/Layout/Footer/Footer.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { useInView } from 'framer-motion';
 import { Link } from '@app/components/Link';
 import { useFooter } from '@app/hooks/useFooter';
 import { createBemBlockBuilder, isNewYearMode } from '@app/utils';
+import { Certificates } from '@app/components/CertificationCard/Certificates';
```
Comment (ViktorSoroka07) [inline src/components/Layout/Footer/Footer.tsx]:
```suggestion
import { Link } from '@app/components/Link';
import { Certificates } from '@app/components/CertificationCard/Certificates';
import { useFooter } from '@app/hooks/useFooter';
import { createBemBlockBuilder, isNewYearMode } from '@app/utils';
```

File: src/components/CertificationCard/CertificationCard.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <div className={getBlocksWith()}>
+      <h2>ReportPortal certifications</h2>
+      {subtitle && <div className={getBlocksWith('__subtitle')}>{subtitle}</div>}
+      <div className={getBlocksWith('__certificates')}>
+        <Certificates />
+      </div>
+      {shouldDisplayLink && (
+        <ArrowLink
+          text="Learn more"
+          to="https://reportportal.io/blog/reportportal-completes-soc-2-type-ii-audit"
```
Comment (ViktorSoroka07) [inline src/components/CertificationCard/CertificationCard.tsx]: We should not hardcode the `https://reportportal.io` as app can be deployed to other envs
```suggestion
          to="/blog/reportportal-completes-soc-2-type-ii-audit"
```

Will suggested variant work?

File: src/components/CertificationCard/Certificates/Certificates.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+const getBlocksWith = createBemBlockBuilder(['certificates']);
+
+const certificatesIcons = [Soc2, Isae3402, Iso9001, Iso27001];
+
+interface CertificatesProps {
+  className?: string;
+}
+
+export const Certificates: FC<CertificatesProps> = ({ className }) => {
+  return (
```
Comment (ViktorSoroka07) [inline src/components/CertificationCard/Certificates/Certificates.tsx]: Excess `return`

File: src/components/CertificationCard/Certificates/Certificates.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    <div className={cn(getBlocksWith(), className)}>
+      {certificatesIcons.map(Certificate => (
+        <Certificate key={Certificate} />
```
Comment (ViktorSoroka07) [inline src/components/CertificationCard/Certificates/Certificates.tsx]: Will the key here be converted to `[Object object]`?

Comment (yelyzavetakhokhlova) [reply src/components/CertificationCard/Certificates/Certificates.tsx]: No, to the create element function body

**Review by ViktorSoroka07 — APPROVED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**